Data/ni.yaml

```diff
@@ -0,0 +1,47 @@
+meta:
+  name: nickel
+  formula: Ni
+  last_updated: 7272181873150
+  description:
+  comment:
+  references:
+    '@online{wikipedia,
+      author = "Wikipedia",
+      title = "Nickel",
+      url = "https://de.wikipedia.org/wiki/Nickel"
+      }
+
+    @online{ri,
+      author = "refractiveindex.info",
+      title = "Nickel",
+      url = "https://refractiveindex.info/?shelf=main&book=Ni&page=Johnson"
+      }'
+
+data:
+  thermal_conductivity:
+    wikipedia:
+      value: 91
+      units: [W/(mK)]
```

Comment:
I would avoid having the square brackets around the units

Reply:
YES! Thank you!

Reply:
I just removed the parentheses. parser works still fine.